cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -579,7 +579,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: rook-ceph-mgr
-  namespace: rook-ceph
+  namespace: rook-ceph # namespace:cluster
```

Comment:
In multi-cluster condition, this should add identical subject with cluster namespace for each cluster, rather than just replace, which will effect the former created cluster.